tests/backup/backup_test.go

```diff
@@ -60,14 +57,121 @@ func TearDownBackupRestore(bkpNamespaces []string, restoreNamespaces []string) {
 
 //This testcase verifies if the backup pods are in Ready state or not
 var _ = Describe("{BackupClusterVerification}", func() {
+	JustBeforeEach(func() {
+		log.Infof(" No pre-setup required for this testcase")
+	})
 	It("Backup Cluster Verification", func() {
-		Step("Check the status of pxcentral-post-install-hook pods", func() {
+		Step("Check the status of backup pods", func() {
+			dash.Info("Check the status of backup pods")
 			status := validateBackupCluster()
-			Expect(status).NotTo(Equal(false),
-				fmt.Sprintf("Backup pods are not in expected state"))
+			dash.VerifyFatal(status, false, fmt.Sprintf("Backup pods are not in expected state"))
```

Comment:
Fix the fatal condition, I think true is expected here

Reply:
done!!